plugin.php

```diff
@@ -5,21 +5,24 @@
 Description: This is a bridge plugin to make <a href="https://wordpress.org/plugins/wordpress-mu-domain-mapping/" target="_blank">MU Domain Mapping</a> compatible with WPML
 Author: OnTheGoSystems
 Author URI: http://www.onthegosystems.com/
-Version: 1.0.0
+Version: 1.0.1
 Plugin Slug: wpml-mu-domain-mapping
 */
 
+if ( defined( 'WPML_MU_DOMAIN_MAPPING_PATH' ) || get_option( '_wpml_inactive' ) ) {
+	return;
+}
+
 define( 'WPML_MU_DOMAIN_MAPPING_ID', plugin_basename( __FILE__ ) );
+define( 'WPML_MU_DOMAIN_MAPPING_PATH', dirname( __FILE__ ) );
 
 require_once 'classes/class-wpml-mu-domain-mapping-requirements.php';
 new WPML_MU_Domain_Mapping_Requirements();
 
 function wpml_mu_domain_mapping_enable() {
 	global $wpdb, $sitepress;
 
-	$wpml_auto_loader_instance = WPML_Auto_Loader::get_instance();
-	$wpml_auto_loader_instance->register( dirname( __FILE__ ) . '/' );
-
+	require_once 'classes/class-wpml-mu-domain-mapping-filters.php';
```

Comment:
@sciamannikoo we agreed to avoid using composer for non-dev, so they can just download and install it on their site out-of-the-box.